resolvers.ts

```diff
@@ -77,42 +77,43 @@ export const resolvers: Resolvers = {
     },
     SampleMetadata: async (root, args, context, info) => {
       const url = `/samples/${args.sampleId}/metadata`;
-      const urlWithParams = args?.input?.pipelineVersion ? url + `?pipeline_version=${args?.input?.pipelineVersion}` : url;
-      const res = await get(
-        urlWithParams,
-        args,
-        context
-      );
+      const urlWithParams = args?.input?.pipelineVersion
```

Comment:
I know this PR wasn't the original place where these resolvers were created, but any chance we could add some unit tests for the resolvers that it touches (similar to [the ZipLink test](https://github.com/chanzuckerberg/czid-graphql-federation-server/blob/112930adf4dd3921427a794b33dfdc9536bc57a9/tests/zipLinkQuery.test.ts#L12))?  If there is a sample response for these resolvers, we might be able to just parse that and stub the http call response with it.